### PR TITLE
Add "pt_BR" translation files.

### PR DESCRIPTION
**Pull request by allysonbarros**
Brazilian Portuguese translation for extension's messages.

### CODE REVIEW

File: ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,492 @@
...
+
+#: ckanext/datarequests/templates/datarequests/index.html:9
+#: ckanext/datarequests/templates/organization/datarequests.html:10
+msgid "Add Data Request"
+msgstr "Adicionar solicitação de dados"
+
+#: ckanext/datarequests/templates/datarequests/index.html:12
+#: ckanext/datarequests/templates/organization/datarequests.html:13
+#: ckanext/datarequests/templates/user/datarequests.html:9
+msgid "Pesquisar solicitações de dados..."
```
Comment (aitormagan) [inline ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po]: I think there is a mistake here :(

File: ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,492 @@
...
+msgid "groups"
+msgstr "grupos"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:28
+msgid "related item"
+msgstr "item relacionado"
+
+#: ckanext/datarequests/templates/home/snippets/stats.html:28
+msgid "related items"
+msgstr "itens relacionados"
```
Comment (aitormagan) [inline ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po]: itens or items?

Comment (allysonbarros) [reply ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po]: That's correct. In portuguese the plural of "item" is "itens"...

File: ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po (hunk 1 above)
Comment (allysonbarros) [reply ckanext/datarequests/i18n/pt_BR/LC_MESSAGES/pt_BR.po]: Ops, sorry... I'll adjust this line.

**Comment by allysonbarros** [comment]
@aitormagan should I close the MR and open a new one for the `develop` branch?

**Comment by allysonbarros** [comment]
@aitormagan I fixed the files and changed the source branch of this pull request for `develop` branch.

**Comment by aitormagan** [comment]
Chaning the branch is not an option because now the PR includes another commits that should not be in the `develop` branch. The best option is to close this PR an create another one using `develop` as base.

I can do it for you so don't worry about this. However, in order to do that I still need the compiled file. Please can you generate and commit it? 

Thanks you so much for your contribution :) :)

**Pull request closed without merging**